partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py

```diff
@@ -207,44 +198,42 @@ def test_04_contact_creation(self):
         ctx = {'search_show_all_positions': {'is_set': True,
                                              'set_value': True
                                              }}
-        self.partner.unlink(cr, uid, [new_contact_id], context=ctx)
+        self.partner.unlink([new_contact_id])
```

Comment:
You lost the context.
Also, you can directly use `new_contact_id.with_context(...).unlink()`.